docs/build/guides/conventions/upgrading-contracts.mdx

```diff
@@ -74,7 +90,13 @@ pub fn upgrade(e: Env, new_wasm_hash: BytesN<32>) {
 }
 ```
 
-The `update_current_contract_wasm` host function will also emit a `SYSTEM` contract [event] that contains the old and new wasm reference, allowing downstream users to be notified when a contract they use is updated. The event structure will have `topics = ["executable_update", old_executable: ContractExecutable, old_executable: ContractExecutable]` and `data = []`.
+- `e: Env`: The environment object representing the current state of the blockchain.
+- `new_wasm_hash: BytesN<32>`: The hash of the new Wasm code for the contract.
```

Comment:
Can we specify here (or somewhere else? [if it's not already]) that the Wasm bytecode must _already_ be installed/present on the ledger.